packages/pds/src/services/account/index.ts

```diff
@@ -284,6 +290,52 @@ export class AccountService {
     }, {} as Record<string, boolean>)
   }
 
+  async subscribeMuteList(info: {
+    list: string
+    mutedByDid: string
+    createdAt?: Date
+  }) {
+    const { list, mutedByDid, createdAt = new Date() } = info
+    await this.db.db
+      .insertInto('list_mute')
+      .values({
+        listUri: list,
+        mutedByDid,
+        createdAt: createdAt.toISOString(),
+      })
+      .execute()
+  }
+
+  async unsubscribeMuteList(info: { list: string; mutedByDid: string }) {
+    const { list, mutedByDid } = info
+    await this.db.db
+      .deleteFrom('list_mute')
+      .where('listUri', '=', list)
+      .where('mutedByDid', '=', mutedByDid)
+      .execute()
+  }
+
+  mutedQb(requester: string, refs: NotEmptyArray<DbRef>) {
+    const subjectRefs = sql.join(refs)
+    const actorMute = this.db.db
+      .selectFrom('mute')
+      .where('mutedByDid', '=', requester)
+      .where('did', 'in', sql`(${subjectRefs})`)
+      .select('did as muted')
+    const listMute = this.db.db
+      .selectFrom('list_mute')
+      .innerJoin('list', 'list.uri', 'list_mute.listUri')
+      .innerJoin('list_item', (join) =>
+        join
+          .onRef('list_item.creator', '=', 'list.creator')
+          .onRef('list_item.listUri', '=', 'list.uri'),
+      )
+      .where('list_mute.mutedByDid', '=', requester)
+      .whereRef('list_item.subjectDid', 'in', sql`(${subjectRefs})`)
+      .select('list_item.subjectDid as muted')
```

Comment:
I am really sorry if I keep mentioning this creator/listUri thing and it's way off for some reason that I overlooked 😆 

But I think this is another place it might simplify things.  Pretty sure this could become:
```ts
    const listMute = this.db.db
      .selectFrom('list_item')
      .innerJoin('list_mute', 'list_mute.listUri', 'list_item.listUri')
      .where('list_mute.mutedByDid', '=', requester)
      .whereRef('list_item.subjectDid', 'in', sql`(${subjectRefs})`)
      .select('list_item.subjectDid as muted')
```

Reply:
nope this is right & there's another place to fix it as well 👌